packages/kbn-optimizer/limits.yml

```diff
@@ -93,7 +93,7 @@ pageLoadAssetSize:
   expressionShape: 34008
   interactiveSetup: 80000
   expressionTagcloud: 27505
-  securitySolution: 273763
+  securitySolution: 289364 # This is temporary: https://github.com/elastic/kibana/pull/132210 update-limits when feature flag removed and deprecated code cleaned
```

Comment:
@elastic/kibana-operations We are changing the side navigation components, but It is under a feature & uiSettings flag, so we have to maintain both navigations for a while, having duplicated components causes the plugin size to rise significantly.
The size will come back down when the conditional is removed and we are able to clean the old navigation components.
Do we need to do anything else in this situation?

Reply:
I think we are okay for now with that until we can remove it and get the bundles downsized again.